cfme/tests/services/test_service_rbac.py

```diff
@@ -0,0 +1,129 @@
+import fauxfactory
+import pytest
+
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils import error
+
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers'),
+    pytest.mark.usefixtures('setup_provider_modscope'),
+    pytest.mark.provider([VMwareProvider], scope="module")]
+
+
+def new_role(appliance, product_features):
+    collection = appliance.collections.roles
+    return collection.create(name='role_{}'.format(fauxfactory.gen_alphanumeric()),
+                             vm_restriction=None, product_features=product_features)
+
+
+def new_group(appliance, role):
+    collection = appliance.collections.groups
+    return collection.create(description='group_{}'.format(fauxfactory.gen_alphanumeric()),
+                             role=role, tenant="My Company")
+
+
+def new_user(appliance, group, credential):
+    collection = appliance.collections.users
+    return collection.create(name='user_{}'.format(fauxfactory.gen_alphanumeric()),
+                             credential=credential,
+                             email='xyz@redhat.com',
+                             groups=group,
+                             cost_center='Workload',
+                             value_assign='Database')
+
+
+@pytest.yield_fixture(scope="module")
+def role_user_group(appliance, new_credential):
+    role = new_role(appliance=appliance,
+                    product_features=[(['Everything'], False)] + [(['Everything', k], True)
+                                    for k in ['Access Rules for all Virtual Machines']])
+    group = new_group(appliance=appliance, role=role.name)
+    user = new_user(appliance=appliance, group=group, credential=new_credential)
+    yield (role, user)
+
+
+@pytest.yield_fixture(scope='function')
+def _myservice(appliance, catalog_item):
+    vm_name = catalog_item.provisioning_data["catalog"]["vm_name"]
+    catalog_item.create()
```

Comment:
catalog item is created in catalog_item fixture itself . No need to create here.